index.js

```diff
@@ -55,34 +65,43 @@ async function assumeRole(params) {
     tagArray.push({Key: 'Branch', Value: process.env.GITHUB_REF});
   }
 
-  const roleSessionTags = roleSkipSessionTagging ? undefined : tagArray;
+  const roleSessionTags = roleSkipSessionTagging || isDefined(webIdentityTokenFile) ? undefined : tagArray;
```

Comment:
nice catch re: the lack of `Tags` on `assumeRoleWithWebIdentity` - makes sense to omit them in that case, though I'd prefer we handle removing (vs. omitting) the tags key from `assumeRoleRequest` within [the `if` block](https://github.com/aws-actions/configure-aws-credentials/pull/240/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R83) that checks for `webIdentityToken` instead of here. That way we can keep all the web token logic close together and log what actions we're taking at once. E.g.: 

```
core.debug("webIdentityTokenFile provided. Will call sts:AssumeRoleWithWebIdentity and take session tags from token contents.")
```